ansible/roles/csm.rr.ceph_zoning/tasks/main.yml

```diff
@@ -24,15 +24,40 @@
 ---
 # Apply CEPH zoning to make sure CEPH data gets replicated at rack level, 
 # so that there would not be data loss incase of a rack failure.
+
+- name: Check if CEPH rack entries with assigned children exist
+  shell: |
+    ceph osd tree -f json | jq -r '.nodes[] | select(.type == "rack" and (.children | length > 0)) | .name'
+  register: rack_check
+  delegate_to: "{{ play_hosts | first }}"
+  run_once: true
+  changed_when: false
+
+- name: Debug message when CEPH racks with children exist
```

Comment:
If the previous execution attempt created the first rack and then hit a failure, won't this mean that the second attempt will see the one rack and assume it is all done, even though it didn't get to create all of the racks?